docs/develop/go/core-application.mdx

```diff
@@ -437,7 +303,9 @@ The Temporal Go SDK has APIs to handle equivalent Go constructs:
 - `workflow.Context` This is a replacement for `context.Context`.
   See [Tracing](/develop/go/observability#tracing-and-context-propogation) for more information about context propagation.
 
-## How to develop an Activity Definition in Go {#activity-definition}
+## Develop an Activity Definition in Go {#activity-definition}
```

Comment:
You replaced "in Go" with "using the Go SDK" above so maybe you can replace this with 'using the Go SDK' as well